### PR TITLE
[FIRRTL] Add force/release operations to ODS.

### PR DESCRIPTION
**Pull request by dtzSiFive**
Simply add these to the IR, not supported anywhere yet.

### CODE REVIEW

File: test/Dialect/FIRRTL/errors.mlir [hunk 1]
```diff
@@ -1176,6 +1176,30 @@ firrtl.circuit "ForceableTypeMismatch" {
...
 }
 
+// -----
+
+firrtl.circuit "RefForceProbe" {
+  firrtl.module @RefForceProbe() {
+    %a = firrtl.wire : !firrtl.uint<1>
+    %1 = firrtl.ref.send %a : !firrtl.uint<1>
+    // expected-note @above {{prior use here}}
+    // expected-error @below {{use of value '%1' expects different type than prior uses: '!firrtl.rwprobe<uint<1>>' vs '!firrtl.probe<uint<1>>'}}
```
Comment (mikeurbach) [inline test/Dialect/FIRRTL/errors.mlir:1186]: Thanks for adding tests for this. IMHO it's fine to not add error checks for things ODS guarantees, but no problem having more checks.

**Review by mikeurbach — APPROVED** [review]
Makes sense to me.

File: test/Dialect/FIRRTL/errors.mlir (hunk 1 above)
Comment (dtzSiFive) [reply test/Dialect/FIRRTL/errors.mlir:1186]: Yeah, this is less about verifying the details of the diagnostic as much as ensuring /something/ catches this and that things don't crash before producing some sort of relevant message.

**Comment by dtzSiFive** [comment]
Thanks!

**Pull request merged**